tabs/src/translations/en-US.json

```diff
@@ -74,6 +74,14 @@
             "Go to Graph Explorer": "Go to Graph Explorer",
             "Graph Explorer Documentation": "Graph Explorer Documentation",
             "Resource-Specific Consent": "Resource-Specific Consent"
+        },
+
+        "Sample Queries" : {
+            "loading": "Loading Sample Queries"
+        },
+
+        "RSC" : {
+            "loading": "Loading Permissions"
```

Comment:
Can this be switched to "Loading Granted Resource-Specific Consent" or if it looks too long "Loading Granted Permissions"